proto/p4/config/v1/p4info.proto

```diff
@@ -46,6 +48,24 @@ message Documentation {
   string description = 2;
 }
 
+message KVmap {
+  map <string,string> property = 1;   // a map of key-value pairs
+}
+
+// A parsed representation of the annotation body.
+// Ex. empty annotation:   @my_anno1()
+// Ex. string annotation:  @my_anno2("Any text at all", for (i = 1 to 10) {print i} )
+// Ex. KV-pair annotation: @my_anno3(a=b,c=d)
+// NOTE: multiple KV annotations with the same tag are cumulative, so @my_anno3(a=b,c=d) in conjunction with
+//       @my_anno3(e=f) would result in 3 kv-pairs for my_anno3.
+message AnnotationBody {
+  oneof anno {
```

Comment:
what about `body` instead of `anno`?

Reply:
Good idea, but I removed the `oneof` so it's moot.